docs/elasticsearch/catapis.md

```diff
@@ -56,53 +56,68 @@ Typically, for any operation you can find out what headers are available using t
 {:toc}
 
 ---
+## Aliases
+
+Shows the mapping of aliases to indices, plus routing and filtering information.
+
+```json
+GET _cat/aliases?v
+```
+
+To limit the information to a specific alias, add the alias name after your query.
+
+```json
+GET _cat/aliases/<alias>?v
+```
 
 ## Allocation
 
 Shows the allocation of disk space for indices and the number of shards on each node.
-
+Default request:
 ```json
 GET _cat/allocation?v
 ```
+Example default response:
```

Comment:
I like the idea of including at least one sample response on this page, but I think I'd rather see it right at the top so that people get an idea of what these APIs return. What do you think about moving this right below the second paragraph with a little, "For example, a request to `_cat/allocation` might return:"?